tests/src/pod-smoke.spec.ts

```diff
@@ -0,0 +1,177 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Page } from 'playwright';
+import type { RunnerTestContext } from './testContext/runner-test-context';
+import { afterAll, beforeAll, test, describe, beforeEach, expect } from 'vitest';
+import { expect as playExpect } from '@playwright/test';
+import { PodmanDesktopRunner } from './runner/podman-desktop-runner';
+import { WelcomePage } from './model/pages/welcome-page';
+import { NavigationBar } from './model/workbench/navigation';
+import { waitUntil, waitWhile } from './utility/wait';
+import { deleteContainer, deleteImage, deletePod } from './utility/operations';
+import { ContainerState, PodState } from './model/core/states';
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+
+const imageToPull = 'docker.io/library/nginx';
+const imageTag = 'latest';
+const containerToRun = 'nginx-container';
+const podToRun = 'nginx-pod';
```

Comment:
For the podify scenario maybe I would use the images of demo that we're doing

it contains a python container and redis so I would assume that what we test is what we're demoing

for example nginx doesn't work on OpenShift clusters, etc.

https://quay.io/repository/podman-desktop-demo/podify-demo-frontend
and
https://quay.io/repository/podman-desktop-demo/podify-demo-backend

Reply:
I would agree to do it as a part of a follow up issue. We would also need to update POM of the containers creation form and probably add another ARIA labels. WDYT?

Reply:
Yes can be a follow up

Reply:
@benoitf  Issue for ARIA functionality - https://github.com/containers/podman-desktop/issues/4278

Reply:
Here is a link to the follow-up issue:
https://github.com/containers/podman-desktop/issues/4279#issue-1937640594